testing/test_mark.py

```diff
@@ -891,3 +891,26 @@ class TestMarkDecorator(object):
     ])
     def test__eq__(self, lhs, rhs, expected):
         assert (lhs == rhs) == expected
+
+
+@pytest.mark.parametrize('mark', [None, 'skip', 'xfail'])
```

Comment:
You check for the empty string in the code, should you add an empty string here too I suppose?